compression/peak_signal_to_noise_ratio.py

```diff
@@ -8,36 +8,54 @@
 import math
 import os
 
-import cv2
-import numpy as np
-
 PIXEL_MAX = 255.0
 
 
-def peak_signal_to_noise_ratio(original: float, contrast: float) -> float:
-    mse = np.mean((original - contrast) ** 2)
+def peak_signal_to_noise_ratio(original, contrast):
```

Comment:
Please revert the changes to this line. Removing the type hints is NOT allowed.  That would be going in the wrong direction since it would be blinding `mypy` which runs in our GitHub Actions.

Modifying functions that have no tests is also NOT allowed.  Please create a separate pull request that adds doctests.  Once that pull request is merged then we can come back to this pull request.